runtime/lua/vim/lsp/rpc.lua

```diff
@@ -608,6 +619,83 @@ function M.connect(host, port)
   end
 end
 
+---@alias vim.lsp.rpc.Dispatcher fun(method: string, params: table<string, any>)
+---@alias vim.lsp.rpc.on_error fun(code: integer, ...: any)
+---@alias vim.lsp.rpc.on_exit fun(code: integer, signal: integer)
+
+---@class vim.lsp.rpc.PrivateDispatchers
+---@field notification vim.lsp.rpc.Dispatcher
+---@field server_request vim.lsp.rpc.Dispatcher
+---@field on_exit vim.lsp.rpc.on_error
+---@field on_error vim.lsp.rpc.on_exit
+
+-- TODO: better name
+---@class vim.lsp.rpc.StartServer
+---@field request fun(method: string, params: table?, callbackfun: fun(err: lsp.ResponseError | nil, result: any), notify_reply_callback:function?)
```

Comment:
```suggestion
---@field request fun(method: string, params: table?, callback: fun(err?: lsp.ResponseError, result: any), notify_reply_callback:function?)
```

It's usually just `callback`:

```
↪  rg callback **/*.lua | wc -l
569
```


```
↪  rg callbackfun **/*.lua | wc -l
0
```